app/views/navigation/_life_stages.html.erb

```diff
@@ -0,0 +1,195 @@
+                    <% NavigationHelper::LIFE_STAGES_RETIREMENT.each do |text, link| %>
+                      <%= render_level_3_subcategory_link(text, link) %>
+                    <% end %>
+                  </ul>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </li>
+
+        <li class="nav__level-2__category" role="none">
+          <%= render_level_2_extended_heading('People in Financial Difficulties') %>
+
+          <div class="nav__level-3" data-nav-level-3>
+            <div class="nav__level-3__main">
+              <div class="nav__level-3__image"></div>
+
+              <p class="nav__level-3__description">People can experience financial difficulties at any stage in life. This section covers early intervention, crisis support and building resilience to future financial shocks.</p>
+
+              <%= render_level_3_heading('People in Financial Difficulties') %>
+
+              <ul class="nav__level-3__categories" role="menu">
+                <%= render_level_3_subcategory_btn('Find out more about people in financial difficulties', lifestage_path('en', 'people-financial-difficulties')) %>
+
+                <li class="nav__level-3__category" role="none">
+                  <ul class="nav__level-3__subcategories" role="menu" aria-label="People in Financial difficulties">
+                    <% NavigationHelper::LIFE_STAGES_FINANCIAL_DIFFICULTIES.each do |text, link| %>
+                      <%= render_level_3_subcategory_link(text, link) %>
+                    <% end %>
+                  </ul>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </li>
+
+        <li class="nav__level-2__category" role="none">
+          <%= render_level_2_extended_heading('People trying to save') %>
+
+          <div class="nav__level-3" data-nav-level-3>
+            <div class="nav__level-3__main">
+              <div class="nav__level-3__image"></div>
+
+              <p class="nav__level-3__description">As a nation, very few of us have enough savings. This section looks at ways to tackle this</p>
+
+              <%= render_level_3_heading('People trying to save') %>
+
+              <ul class="nav__level-3__categories" role="menu">
+                <%= render_level_3_subcategory_btn('Find out more about people trying to save', lifestage_path('en', 'people-trying-save')) %>
+
+                <li class="nav__level-3__category" role="none">
+                  <ul class="nav__level-3__subcategories" role="menu" aria-label="People trying to save">
+                    <% NavigationHelper::LIFE_STAGES_SAVING.each do |text, link| %>
+                      <%= render_level_3_subcategory_link(text, link) %>
+                    <% end %>
+                  </ul>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+</li>
```

Comment:
I'd need to look further into this but I would have thought that we could be making use of the https://github.com/moneyadviceservice/fin_cap/blob/master/config/locales/en.yml#L62-L76